### PR TITLE
Release 6.3.0

### PR DESCRIPTION
**Pull request by AliSoftware**
Let's get those new stringsdict features out!

_This release PR targets the `stable` branch. To just review the changes since the release PR was cut from `develop` branch, see [here](https://github.com/SwiftGen/SwiftGen/compare/develop...release/6.3.0)_

### CODE REVIEW

**Comment by SwiftGen-Eve** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should make sure that <code>Podfile.lock</code> and <code>Package.resolved</code> are changed in sync</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This is a Release PR</td>
    </tr>
  </tbody>
</table>

Hey 👋 I'm Eve, the friendly bot watching over SwiftGen 🤖

Thanks a lot for your contribution!



---

### ChangeLog check

✅  All entries seems OK (end with period + 2 spaces, correct links)


### Release version check

| Check                                         | Status       |
| --------------------------------------------- | ------------ |
| Bundler installed                             | ✅           |
| SwiftGen.podspec                              | 👉  6.3.0    |
| SwiftGenKit.podspec                           | 👉  6.3.0    |
| SwiftGen & SwiftGenKit versions equal         | ✅           |
| SwiftGen-Info.plist version matches           | ✅           |
| SwiftGenKit-Info.plist version matches        | ✅           |
| StencilSwiftKit up-to-date (latest: 2.7.2)    | ✅           |
| CHANGELOG: Release entry added                | ✅           |
| CHANGELOG: No develop entry                   | ✅           |
| README: pod version ~> 6.0                    | ✅           |


Once you fix those tiny nitpickings above, we should be good to go! 🙌

ℹ️ _I will update this comment as you add new commits_

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by djbe — CHANGES_REQUESTED** [review]
Changes here all look OK, a couple of things we may want to update while we're creating a release:
- [x] Mint installation instructions: remove `brew install libxml2`
- [x] Changelog entries for `bundle` & `lookupFunction`: point to dedicated article about these.
- [x] Maybe a small template migration guide entry for the font typealias deprecation? We did write a tiny entry in contexts for the `documents` change in 6.2.1

**Review by djbe — APPROVED** [review]

File: Documentation/templates/MigrationGuide.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# SwiftGen 6.3 Templates Migration Guide
+
+There are no major changes in the templates, only a small deprecation notice:
+
+In the `fonts` templates, we're deprecating the generated `Font` typealias. So if you were using it somewhere in your own code, create your own typealias, or use the platform sepcific type such as `UIFont`/`NSFont`. The `Font` typealias will be removed in the next major release of SwiftGen.
```
Comment (AliSoftware) [inline Documentation/templates/MigrationGuide.md]:
```suggestion
In the `fonts` templates, we're deprecating the generated `Font` typealias. So if you were using it somewhere in your own code, you should instead create your own typealias, or use the platform-specific type such as `UIFont`/`NSFont`. The `Font` typealias will be removed in the next major release of SwiftGen.
```

**Comment by AliSoftware** [comment]
```
--------------------------------------------------------------------------------
 :tada:  Congrats

 :rocket:  SwiftGen (6.3.0) successfully published
 :date:  August 5th, 00:00
 :earth_americas:  https://cocoapods.org/pods/SwiftGen
 :+1:  Tell your friends!
--------------------------------------------------------------------------------
```

**Comment by AliSoftware** [comment]
Pending PR in Homebrew: https://github.com/Homebrew/homebrew-core/pull/59153

**Pull request merged**